CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ option(DISABLE_RCT2 "WIP: Try building without using code and data segments from
 option(USE_MMAP "Use mmap to try loading rct2's data segment into memory.")
 option(LAUNCHPAD_BUILD "Sets some build system options on launchpad.net")
 
-set(COMMON_COMPILE_OPTIONS "${COMMON_COMPILE_OPTIONS} -fstrict-aliasing -Werror -Wundef -Wmissing-declarations")
+set(COMMON_COMPILE_OPTIONS "${COMMON_COMPILE_OPTIONS} -fstrict-aliasing -Werror -Wundef -Wmissing-declarations -Winit-self -Warray-bounds -Waddress -Wchar-subscripts -Wenum-compare")
```

Comment:
Thank you for your quick response.

Sorry. I've neglected to mention that my last complete system update was last week. I'm stuck with a 7 kB/s connection at the moment. 

I will remove the `-Warray-bounds` locally until I'm back at home. I'm curious if this affects other people as well. gcc 6.2.1 only made it into core/multlib last Friday…